conda_forge_tick_action/automerge.py

```diff
@@ -166,14 +169,19 @@ def _automerge_pr(repo, pr, session):
     if '[bot-automerge]' not in pr.title:
         return False, "PR does not have the '[bot-automerge]' slug in the title"
 
+    return True, None
+
+
+def _automerge_pr(repo, pr, session):
+    allowed, msg = _check_pr(pr)
+
+    if not allowed:
+        return False, msg
+
     # now load the the conda-forge config
     with open(os.path.join(os.environ['GITHUB_WORKSPACE'], 'conda-forge.yml')) as fp:
         cfg = YAML().load(fp)
 
-    # can we automerge in this feedstock?
-    if not _automerge_me(cfg):
-        return False, "automated bot merges are turned off for this feedstock"
-
```

Comment:
We still want this check for the case with the pr title.

Reply:
Would the bot send PRs for feedstocks without this?

Reply:
Idk but that doesn’t matter. We always want any feedstock to be able to opt out if they want to. This we need this check.